api/external_endpoint_test.go

```diff
@@ -138,16 +150,19 @@ func TestCreateExternalEndpointResult(t *testing.T) {
 		if len(endpointStatus.Results[1].Errors) > 0 {
 			t.Errorf("expected second result to have no errors")
 		}
-		if endpointStatus.Results[2].Success {
-			t.Errorf("expected third result to be unsuccessful")
```

Comment:
I don't think the numbering makes much sense here.  The `[2]` element is still the third, not the sixth